tools/accuracy_checker/configs/mosaic.yml

```diff
@@ -0,0 +1,35 @@
+models:
+  - name: style_transfer
+
+    launchers:
+
+      - framework: onnx_runtime
+        device: CPU
+        model: public/mosaic/mosaic.onnx
+        adapter: style_transfer
+
+      - framework: dlsdk
+        device: CPU
+        model: public/mosaic/FP32/mosaic.xml
+        weights: public/mosaic/FP32/mosaic.bin
+        adapter: style_transfer
```

Comment:
it is also will be cool if you provide section for FP16 model validation

Reply:
FP16 model added